src/enum_net.cpp

```diff
@@ -406,14 +374,12 @@ namespace {
 #pragma clang diagnostic pop
 #endif
 
-		auto ifi_info = link_info.find(int(addr_msg->ifa_index));
-		if (ifi_info != link_info.end())
+		if (ip_info->name[0] == '\0')
 		{
-			ip_info->mtu = ifi_info->second.mtu;
 			// for some reason IPv6 entries don't include an IFA_LABEL attribute
 			// so get it from the link in that case
```

Comment:
this comment may be slightly outdated now. I also wonder if it would make sense to use this path to get the device name unconditionally, and not bother parsing the ``IFA_LABEL`` message.